network/p2p/connManager.go

```diff
@@ -79,7 +79,10 @@ func (c ConnManager) updateConnectionMetric(n network.Network) {
 
 func (c ConnManager) logConnectionUpdate(n network.Network, con network.Conn, logMsg string) {
 	c.log.Debug().
-		Str("remote_peer", con.RemoteMultiaddr().String()).
+		Str("remote_peer", con.RemotePeer().String()).
```

Comment:
just addr wasn't superhelpful while debugging hence logging some more information when a connection is established.